### DISCUSSION TITLE
Kubernetes Ingress Prefix Configuration

### DISCUSSION THREAD

**Discussion by pablocuestagarcia** [getredash/redash#6942] [Q&A]
I have deployed Redash on Kubernetes using the Redash Helm chart, and I would like to ask about the Ingress configuration. Due to restrictions I cannot use a subdomain right now, so I am using a path to redirect traffic to the redash service via Ingress. The problem is in the application redirects. I have explored the redash configuration and did not see that there is a way to tell the application that it is located on a subpath. Is this the case? Is a subdomain needed to be able to configure redash correctly?
Best regards and thanks!

**Comment by kongsiyu** [comment]
same question
